### ISSUE TITLE
docs: add a list of configurable keys in README

### ISSUE THREAD

**Issue by kohane27** [farzadmf/tmux-tilish#1]
Hello @farzadmf !

Thank you for creating this fork; it's exactly what I need. I need to disable a lot of the per-configured keybindings because they're conflicting with my nvim keybindings.

Could you please add a list of keys that are configurable? I looked into the src and for example, having trouble disabling the following:

```sh
# Reload configuration with Alt + Shift + c.
tmux $bind "${mod}C" \
	source-file ~/.tmux.conf \\\;\
	display "Reloaded config"
# }}}
```

What's the name for the above "reload configuration" keybinding?

It'd be helpful if you could fill in the following table and add it to README:

| Description                                          | Name             | Key |
| ---------------------------------------------------- | ---------------- | --- |
| Switch to workspace number 0-9                       |                  |     |
| Move pane to workspace 0-9                           |                  |     |
| Move focus left/down/up/right                        |                  |     |
| Move pane left/down/up/right                         |                  |     |
| Create a new pane at "the end" of the current layout |                  |     |
| Switch to layout: split then vsplit                  |                  |     |
| Switch to layout: only split                         |                  |     |
| Switch to layout: vsplit then split                  |                  |     |
| Switch to layout: only vsplit                        |                  |     |
| Switch to layout: fully tiled                        |                  |     |
| Switch to layout: zoom (fullscreen)                  |                  |     |
| Refresh current layout                               | `tilish-refresh` | `r` |
| Name current workspace                               | `tilish-rename`  | `n` |
| Quit (close) pane                                    |                  |     |
| Exit (detach) `tmux`                                 |                  |     |
| Reload config                                        |                  |     |

Thank you again:)

**Comment by farzadmf** [comment]
Hey @kohane27 , thank you for your first issue here 😝 

I really like the idea of having a table in the `README`, I'll look into it and add it to hopefully very soon

**Comment by farzadmf** [comment]
Hey @kohane27 , it's been a while that I changed this guy. I started taking a look, and I see there's already [a table in the `README`](https://github.com/farzadmf/tmux-tilish#keybindings) listing the keybindings (default ones)

Also, in the `README`, it mentions how to change/disable those, for example [layout keys](https://github.com/farzadmf/tmux-tilish#keybindings) section mentions how to configure the layout keys.

Also, as an example, the reload config binding is also mentioned in the table above

**Comment by farzadmf** [comment]
@kohane27  I'm gonna close this for now then. Feel free please to open a new one (or comment here) in case you think there's something I can do for this

**Comment by kohane27** [comment]
hello @farzadmf ! Sorry it took me some time to get back to you; this issue isn't very critical and it took a backseat for some time. My apologies.

## TL;DR: what's the name for the following three empty options?

| Description                                          | Name                     | Key |
| ---------------------------------------------------- | ------------------------ | --- |
| Refresh current layout                               | `tilish-refresh`         | `r` |
| Name current workspace                               | `tilish-rename`          | `n` |
| Quit (close) pane                                    |                          |     |
| Exit (detach) `tmux`                                 |                          |     |
| Reload config                                        |                          |     |

I've looked into the source:

```sh
# Close a window with Alt + Shift + q.
if [ -z "$legacy" ]
then
	tmux $bind "${mod}Q" \
		if-shell \
			'[ "$(tmux display-message -p "#{window_panes}")" -gt 1 ]' \
			'kill-pane; select-layout; select-layout -E' \
			'kill-pane'
else
	tmux $bind "${mod}Q" \
		kill-pane
fi
```

```sh
# Close a connection with Alt + Shift + e.
tmux $bind "${mod}E" \
	confirm-before -p "Detach from #H:#S? (y/n)" detach-client
```

```sh
# Reload configuration with Alt + Shift + c.
tmux $bind "${mod}C" \
	source-file ~/.tmux.conf \\\;\
	display "Reloaded config"
```

Seems like for these 3 options, there are no ways to disable them, except commenting them out in src. Am I correct? Not that I need to remap these 3 keymaps, but hopefully this reply provides pointers for others who do need them.

## How to disable the layout_keys

Given the default `tilish-layout_keys 'sSvVtz'`, each key corresponds to the following list:

```
Alt + s 	        Switch to layout: split then vsplit
Alt + Shift + s 	Switch to layout: only split
Alt + v 	        Switch to layout: vsplit then split
Alt + Shift + v 	Switch to layout: only vsplit
Alt + t 	        Switch to layout: fully tiled
Alt + z 	        Switch to layout: zoom (fullscreen)
```

To disable certain layouts, input `-` in its place. For example, `set -g @tilish-layout_keys '----tz'` means that only tiled and zoom layouts are remained.
I've disabled `Alt+s` and `Alt+v` because I need them for kitty/nvim keymappings, so `set -g @tilish-layout_keys 'S-V-tz'`
Hopefully this section is even more clear for others:)

**Comment by farzadmf** [comment]
> there are no ways to disable them, except commenting them out in src. Am I correct?

@kohane27 I need to double check to be absolutely sure, but as far as I remember, yes, there's no way to disable those

I mostly added the ones the I felt the need to configure/disable, and it maybe indeed the case that I didn't need to do any modifications to those, and didn't add them

**Comment by kohane27** [comment]
@farzadmf 

Please don't bother. I'm very happy with being able to remap the layout keys. If someone requests it, we may look into it together by then.

Thank you again:)

**Comment by farzadmf** [comment]
> If someone requests it, we may look into it together by then

Sounds like a plan 😆
